lib/xcpretty/parser.rb

```diff
@@ -181,6 +181,13 @@ module Matchers
     # $2 file_name
     TOUCH_MATCHER = /^Touch\s(.*\/([\w+\.]+))/
 
+    # @regex Captured groups
+    # $1 file_path
+    WRITE_FILE_MATCHER = /^write-file\s(.*)/
+
+    # @regex Captured groups
+    WRITE_AUXILIARY_FILE = /^Write auxiliary files/
```

Comment:
nitpick: s/WRITE_AUXILIARY_FILE/WRITE_AUXILIARY_FILES

It's a plural, singular made me think there should be a filename parameter